src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyFlowGraphOperation.cs

```diff
@@ -78,9 +80,20 @@ public override async Task<int> ExecuteAsync()
                     }
                 }
 
+                JsonSerializer serializer = new JsonSerializer();
+
+                using (StreamWriter sw = new StreamWriter(@"D:\michelm\input.json"))
+                using (JsonWriter writer = new JsonTextWriter(sw))
+                {
+                    serializer.Serialize(writer, flowGraph);
+                }
+
+
```

Comment:
Looks like test code that is stuck in here.

Reply:
oops!